src/utils/utils.ts

```diff
@@ -0,0 +1,13 @@
+export interface Body {
```

Comment:
You can get the User type from prisma like this. Please give it a try `import {  User } from '@prisma/client';`